opensrp-anc/src/test/java/org/smartregister/anc/library/interactor/YamlConfigInteractorTest.java

```diff
@@ -230,7 +224,7 @@ public void testFetchUpcomingContactsWithNoReferralContactsAndContactSchedule()
         PowerMockito.mockStatic(Utils.class);
 
         PowerMockito.when(PatientRepositoryHelper.getWomanProfileDetails(baseEntityId)).thenReturn(details);
-        PowerMockito.when(AncLibrary.getInstance()).thenReturn(AncLibrary);
+        PowerMockito.when(org.smartregister.anc.library.AncLibrary.getInstance()).thenReturn(AncLibrary);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.anc.library.AncLibrary.getInstance' due to existing import 'org.smartregister.anc.library.AncLibrary'](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=4809639)